### PR TITLE
feat(legacy-grid): add v3 of bootstrap grid - INNO-507

### PR DESCRIPTION
**Pull request by kalinchernev**
For backwards compatibility reasons.
Please test well and provide constructive feedback.

### CODE REVIEW

File: framework/layout/grid/grid/grid.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  title: 'Grid - ECL',
```
Comment (yhuard) [inline framework/layout/grid/grid/grid.config.js]: `ECL Grid` sounds better, doesn't it? :)

File: fractal.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ fractal.components.set('statuses', {
   planned: {
     label: 'Planned',
     description: 'Do not implement.',
-    color: '#FF3333',
+    color: '#337ab7',
```
Comment (yhuard) [inline fractal.js]: Not sure about the change in the colors. Maybe we can discuss it today

File: framework/index.scss [hunk 3]
```diff
@@ -48,7 +48,8 @@
 @import 'components/top-images/top-images';
 
 // Layout
-@import 'layout/grid/grid';
+@import 'layout/grid/grid/grid';
```
Comment (yhuard) [inline framework/index.scss]: `grid/grid/grid` 🎉

File: framework/layout/grid-legacy/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Legacy Bootstrap Grid
```
Comment (yhuard) [inline framework/layout/grid-legacy/README.md]: You can remove this file

File: framework/layout/grid/grid-legacy/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@ec-europa/legacy-bootstrap-grid",
```
Comment (yhuard) [inline framework/layout/grid/grid-legacy/package.json]: Please rename the package `@ec-europa/ecl-***`

**Comment by kalinchernev** [comment]
Review points implemented.

File: framework/index.scss (hunk 3 above)
Comment (kalinchernev) [inline framework/index.scss]: Yeah ... 
I don't mind making 1 index file for the grid which will be referenced in the main index file, though when we have this split is indeed irritating and more obvious that it's easy to remove the legacy.
Anyway, I can change.

**Pull request merged**